src/Microsoft.Health.Dicom.Metadata/Registration/Experimental/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,59 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Azure.Core;
+using Azure.Identity;
+using Azure.Storage.Blobs;
+using EnsureThat;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Health.Blob.Configs;
+using Microsoft.Health.Dicom.Metadata;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static IBlobServiceBuilder AddAzureBlobServiceClient(
+            this IServiceCollection services,
```

Comment:
IBlobServiceBuilder instead of IServiceCollection?

Reply:
I feel leaving only configuration setup in the first build function and move other logic into BlobServiceBuilderExtensions is more clean, but I'm OK with leaving  the logic to create BlobServiceClient here now , since both WebServer and AzureFunction need it.

Reply:
I think the only problem with leaving it out would be the possibility that it's never called (and presumably it's needed by anyone configuring it)